models/dojo.py

```diff
@@ -139,72 +141,38 @@ def print_unallocated(self, unallocated_file=None):
-                            office_rooms[new_room_name].max_occupants):
-                        if self.persons[person_id].office_space_allocated:
-                            old_room_name = self.office_rooms[
-                                self.persons[person_id].office_space_allocated]
-                            old_room_name.occupants.remove(
-                                self.persons[person_id])
-
-                            self.persons[
-                                person_id].office_space_allocated = self.\
-                                office_rooms[new_room_name].room_name
-                            self.office_rooms[new_room_name].occupants.\
-                                append(self.persons[person_id])
-                        else:
-                            self.persons[person_id].office_space_allocated =\
-                                self.office_rooms[new_room_name].room_name
-                            self.office_rooms[new_room_name].occupants.\
-                                append(self.persons[person_id])
-
-                        print("identifier {0} has been reallocated to the office {1}"
-                              .format(person_id, self.persons[person_id].
-                                      office_space_allocated))
-                    else:
-                        print("{} is full".format(new_room_name))
-                else:
-                    if (len(self.living_rooms[new_room_name].occupants) < self.
-                            living_rooms[new_room_name].max_occupants):
-                        if self.persons[person_id].living_space_allocated:
-                            old_room_name = self.living_rooms[
-                                self.persons[person_id].living_space_allocated]
-                            old_room_name.occupants.remove(
-                                self.persons[person_id])
-
-                            self.persons[person_id].living_space_allocated =\
-                                self.living_rooms[new_room_name].room_name
-                            self.living_rooms[new_room_name].occupants.\
-                                append(self.persons[person_id])
-                        else:
-                            self.persons[person_id].living_space_allocated =\
-                             self.living_rooms[new_room_name].room_name
-                            self.living_rooms[new_room_name].occupants.\
-                                append(self.persons[person_id])
-                        print("identifier {0} has been reallocated to the livingroom {1}"
-                              .format(person_id, self.persons[person_id].
-                                      living_space_allocated))
-                    else:
-                        print("{} is full".format(new_room_name))
+    def get_members_of_a_room(person, room_name):
+        if person in self.get_room_occupants:
+            return True
```

Comment:
if we have this as `return person in self.get_room_occupants` we will have catered for both the `True` and the `False` return possibilities..